ecommerce_worker/configuration/local.py

```
@@ -0,0 +1,28 @@
+from logging.config import dictConfig
+
+from ecommerce_worker.configuration.base import *
+from ecommerce_worker.configuration.logger import get_logger_config
+
+
+# CELERY
+BROKER_URL = 'amqp://'
+CELERY_RESULT_BACKEND = 'amqp://'
+# END CELERY
+
+
+# ORDER FULFILLMENT
+ECOMMERCE_API_ROOT = 'http://localhost:8002/api/v2/'
+# END ORDER FULFILLMENT
+
+
+# LOGGING
+logger_config = get_logger_config(debug=True, dev_env=True, local_loglevel='DEBUG')
+dictConfig(logger_config)
+# END LOGGING
+
+
+# Apply any developer-defined overrides.
+try:
+    from .private import *  # pylint: disable=import-error
+except ImportError:
+    pass
```

Log a warning. In the past week I've seen two instances of swallowing this error leading to wasted debug time.